services/bettersocial/fcmToken/sendCommentNotification.js

```diff
@@ -17,11 +17,12 @@ const sendCommentNotification = async (userTargetId, commentAuthor, message, act
     data: {
       feed_id: activity_id,
       type: 'feed'
-    }
+    },
+    token: userTargetToken?.token
   };
   if (userTargetToken) {
     messaging()
-      .sendToDevice(userTargetToken?.token, payload)
+      .send(payload)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for `userTargetToken` before sending the message is good, but the promise returned by `messaging().send(payload)` is not being handled properly. The `.then(() => {})` is essentially swallowing any errors that might occur during the sending of the message. It would be better to handle these errors properly, either by logging them or by allowing them to propagate up the call stack.

```diff
-      .then(() => {});
+      .catch((error) => {
+        console.error(`Failed to send notification: ${error}`);
+      });
```
This way, if there's an error while sending the notification, it won't go unnoticed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->